app/components/vis-bug/vis-bug.element.js

```diff
@@ -3,7 +3,7 @@ import hotkeys    from 'hotkeys-js'
 
 import {
   Handles, Handle, Label, Overlay, Gridlines, Corners,
-  Hotkeys, Metatip, Ally, Distance, BoxModel, Grip
+  Hotkeys, Metatip, Ally, Distance, BoxModel, Grip, Rotation
```

Comment:
```suggestion
  Hotkeys, Metatip, Ally, Distance, BoxModel, Grip, Rotation, Delete
```
?

Reply:
yeah, importing it here would allow you to remove the import of it from features/selectable.js, and also keep all the imports centralized 👍🏻